addons/account/models/account_journal.py

```diff
@@ -446,28 +485,6 @@ def _check_auto_post_draft_entries(self):
     def _onchange_type(self):
         self.refund_sequence = self.type in ('sale', 'purchase')
 
-    def _get_alias_values(self, type, alias_name=None):
```

Comment:
I removed this method as it is never inherited, and it is easy to customize the values in the `create`/`write` of `mail.alias` is needed anyways, and it was fragmenting the code for not so long methods anyway.